ibtk/include/ibtk/PETScLevelSolver.h

```diff
@@ -240,6 +240,16 @@ class PETScLevelSolver : public LinearSolver
      */
     void addLinearOperator(Mat& op);
 
+    /*!
```

Comment:
This function (along with addLinearOperator) does not seem to be used anywhere except in the Stokes solver.

This stuff doesn't belong in the PETScLevelSolver interface unless it is used by all implementations.